microcosm_flask/conventions/crud.py

```diff
@@ -288,8 +288,7 @@ def configure_update(self, ns, definition):
         @wraps(definition.func)
         def update(**path_data):
             headers = dict()
-            # NB: using partial here means that marshmallow will not validate required fields
-            request_data = load_request_data(definition.request_schema, partial=True)
+            request_data = load_request_data(definition.request_schema)
```

Comment:
I'm not sure why we wouldn't want to validate required fields on update; this is for the request itself, and if we specified required fields there, we should be respecting that.

Reply:
I have no idea. It seems intentional, but don't remember why.

Reply:
I dug down to the [original commit/PR](https://github.com/globality-corp/microcosm-flask/pull/7/files#diff-d021575c22812976dcde670b51504493R145) for this file and I couldn't find any comment related to this.